starter-code/src/app/component/log-form-component/log-form-component.component.html

```diff
@@ -0,0 +1,34 @@
+<p>
+  log-form-component works!
+</p>
+
+<div>
+  <form (ngSubmit)="addAccessItem(form)" #form="ngForm" novalidate>
+    <div class="field" [ngClass]="{'has-error': feedbackEnable && nameField.errors}" >
+      <label for="name">Name</label>
+      <input type="text" [(ngModel)]="name" name="name" #nameField="ngModel" required minlength="4" [disabled]="processing">
+      <div *ngIf="feedbackEnabled && nameField.errors">
+        <p *ngIf="nameField.errors.required" class="validation">Name is required</p>
+        <p *ngIf="nameField.errors.minlength" class="validation">Name is too short</p>
+      </div>
+    </div>
+    
+    <div class="field" [ngClass]="{'has-error': feedbackEnable && messageField.errors}">
+      <label for="message">Message</label>
+      <input type="text" [(ngModel)]="message" name="message" #messageField="ngModel" required minlength="10" [disabled]="processing">
+      <div *ngIf="feedbackEnabled && messageField.errors">
+        <p *ngIf="messageField.errors.required" class="validation">Message is required</p>
+        <p *ngIf="messageField.errors.minlength" class="validation">Message is too short</p>
+      </div>
+    </div>
+    <button (click)="showMessage()">Submit</button>
```

Comment:
Why are you binding a method to the click when you have a ngSubmit in the form (line 6).